duckduckgo/macos-browser
# Fix back navigation back/forward arrows

Task/Issue URL: https://app.asana.com/0/1177771139624306/1203281165527755/f

**Description**:
This fixes a regression introduced on https://github.com/more-duckduckgo-org/macos-browser/pull/794 
Only the tab overflow menu should have the new < and > arrows. Tab navigation arrows should still have the 
<- and ->  design

**Steps to test this PR**
1. Create multiple tabs until you see the overflow menu. The arrows should look like this:2. 
<img width="38" alt="image" src="https://user-images.githubusercontent.com/782316/199479311-43e6c42f-e552-462e-8f0e-faa4f19c9500.png">
2. Check the navigation arrows. They should look like this:
<img width="65" alt="image" src="https://user-images.githubusercontent.com/782316/199479411-62186b92-a26a-449a-8d9c-e7a6a000ac28.png">
3. Make sure the disable/enable state are working
4. Test on light/dark more

**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
